Dockerfile

```
@@ -0,0 +1,9 @@
+FROM python:2.7-slim
+
+WORKDIR /sdk
+COPY . /sdk
+
+RUN pip install --upgrade pip \
+    && pip install pytest -r requirements.txt
```

it would be wise to first copy the `requirements.txt`, install reqs, and then copy the rest of the code:

```suggestion
WORKDIR /sdk
COPY ./requirements.txt /sdk

RUN pip install --upgrade pip \
    && pip install pytest -r requirements.txt

COPY . /sdk
```

That way, if you change any files, the requirements won't have to be reinstalled each time.